### PR TITLE
Update process_commit.py

### PR DESCRIPTION
**Pull request by albanD**
Update new release note labelling system message on PRs

### CODE REVIEW

**Review by malfet — APPROVED** [review]

File: .github/scripts/process_commit.py [hunk 1]
```diff
@@ -71,9 +71,13 @@ def get_repo_labels() -> List[str]:
 
 def post_pytorch_comment(pr_number: int, merger: str) -> Any:
     message = {'body' : f"Hey {merger}. \
-    You merged this PR, but no release notes category and topic labels were added. \
-    The list of valid release and topic labels is available \
-    https://github.com/pytorch/pytorch/labels?q=release+notes+or+topic"}
+    You merged this PR, but no it does not have both a 'release notes: ...' and 'topics: ...' label. \
```
Comment (malfet) [inline .github/scripts/process_commit.py:74]: Nit: most of PRs are not merged, but committed
```suggestion
    You've committed  this PR, but no it does not have both a 'release notes: ...' and 'topics: ...' label. \
```

File: .github/scripts/process_commit.py [hunk 2]
```diff
@@ -70,10 +70,14 @@ def get_repo_labels() -> List[str]:
...
-    message = {'body' : f"Hey {merger}. \
-    You merged this PR, but no release notes category and topic labels were added. \
-    The list of valid release and topic labels is available \
-    https://github.com/pytorch/pytorch/labels?q=release+notes+or+topic"}
+    message = {'body' : f"""Hey {merger}. \
+    You merged this PR, but no it does not have both a 'release notes: ...' and 'topics: ...' label. \
+    Please add one of each to the PR. The 'release notes: ...' label should represent the part of \
+    PyTorch that this PR changes (fx, autograd, distributed, etc) and the 'topics: ...' label should \
+    represent the kind of PR it is (non-user visible, new feature, bug fix, perf improvement, etc). \
+    The list of valid labels can be found [here](https://github.com/pytorch/pytorch/labels?q=release+notes) \
```
Comment (atalman) [inline .github/scripts/process_commit.py:78]: We should change query. I suggest something like :
https://github.com/pytorch/pytorch/labels?q=mandatorylabel
And add mandatorylabel to description of each required label

**Review by atalman — CHANGES_REQUESTED** [review]

**Review by janeyx99 — APPROVED** [review]
Ah nice, though looks like there might be some unmatched quotes causing lint to be unhappy (though I cannot tell where)

File: .github/scripts/process_commit.py (hunk 2 above)
Comment (atalman) [reply .github/scripts/process_commit.py:78]: This should fix the fact that https://github.com/pytorch/pytorch/labels?page=2&q=release+notes will result in some extra labels like "module: binaries" and cherry-picked

Comment (albanD) [reply .github/scripts/process_commit.py]: From offline discussion, we agreed that the current query is not perfect but we don't really have a strictly better solution for now. Will keep the current for now and we can re-update in the future if we have a new idea.

**Comment by albanD** [comment]
(doing PRs using only the github edit file is hard...)

**Review by atalman — APPROVED** [review]

**Comment by malfet** [comment]
> Merge failed due to PR 72121 does not match merge rules Raised by https://github.com/pytorch/pytorch/actions/runs/1780186678

My "approve" for some reason is reported as "Commented":
```
        "latestReviews": {
          "nodes": [
            {
              "author": {
                "login": "malfet"
              },
              "state": "COMMENTED"
            },
            {
              "author": {
                "login": "atalman"
              },
              "state": "APPROVED"
            },
            {
              "author": {
                "login": "janeyx99"
              },
              "state": "APPROVED"
            }
          ],
          "totalCount": 3
}
```

**Review by malfet — APPROVED** [review]

**Pull request closed without merging**